frappe/public/js/frappe/socketio_client.js

```diff
@@ -1,12 +1,9 @@
+import { io } from "socket.io-client";
 frappe.socketio = {
 	open_tasks: {},
 	open_docs: [],
 	emit_queue: [],
 	init: function(port = 3000) {
-		if (!window.io) {
```

Comment:
Not sure which change causes Line 50 to be touched during assets building but this warning comes up:

```bash
▲ [WARNING] Using direct eval with a bundler is not recommended and may cause problems

    frappe/public/js/frappe/socketio_client.js:50:3:
      50 │       eval(message);
         ╵       ~~~~

  You can read more about direct eval and bundling here:
  https://esbuild.github.io/link/direct-eval
```

ref: https://esbuild.github.io/content-types/#direct-eval

Reply:
resolved with https://github.com/frappe/frappe/pull/17289/commits/81436dc68844586f31634b52483b64a02e66fab2